chromium_src/components/content_settings/core/common/cookie_settings_base.h

```diff
@@ -7,12 +7,15 @@
 #define BRAVE_CHROMIUM_SRC_COMPONENTS_CONTENT_SETTINGS_CORE_COMMON_COOKIE_SETTINGS_BASE_H_
 
 #define BRAVE_COOKIE_SETTINGS_BASE_H                                      \
- private:                                                                 \
+  bool IsEphemeralCookieAccessAllowed(const GURL& url,                    \
+                                      const GURL& first_party_url) const; \
```

Comment:
I think you could eliminate this method by giving the three-argument version a default argument:
```c++
  bool IsEphemeralCookieAccessAllowed(                                    \
      const GURL& url, const GURL& site_for_cookies,                      \
      const base::Optional<url::Origin>& top_frame_origin = base::nullopt) const;   
```

Reply:
I wanted to stay consistent with the original methods for clarity

Reply:
and also when the two argument version is removed that might cause us to miss something